WinRT.Runtime/Projections/ICommand.cs

```diff
@@ -182,6 +203,10 @@ private static unsafe int Do_Abi_CanExecute_2(IntPtr thisPtr, IntPtr parameter,
                 }
                 return 0;
             }
+
+#if !NETSTANDARD2_0
+            [UnmanagedCallersOnly]
```

Comment:
Can we change this pattern and instead define a dummy/placeholder type? Then this attribute can always be added and substantially reduce these preprocessor locations.

```CSharp
#if NETSTANDARD2_0
internal UnmanagedCallersOnlyAttribute : Attribute
{
}
#endif
```

Reply:
Unless the preprocessor is super slow, I'd stick with status quo - the fake attribute is a bit misleading